src/core/src/energy/functions.rs

```diff
@@ -404,14 +421,61 @@ impl Potential for MorsePotential {
 }
 
 impl PairPotential for MorsePotential {
-    fn tail_energy(&self, _: f64) -> f64 {0.0}
-    fn tail_virial(&self, _: f64) -> f64 {0.0}
+    fn tail_energy(&self, _: f64) -> f64 {
+        0.0
+    }
+    fn tail_virial(&self, _: f64) -> f64 {
+        0.0
+    }
 }
 
 impl BondPotential for MorsePotential {}
 impl AnglePotential for MorsePotential {}
 impl DihedralPotential for MorsePotential {}
 
+/// Gaussian potential.
+///
+/// The following expression of Gaussian potential is used: `V(r) = -a *
+/// exp(-b * r^2)` where `a` is the potential depth and `b` is the potential width.
+///
+/// # Examples
+///
+/// ```
+/// use lumol_core::energy::Potential;
+/// use lumol_core::energy::Gaussian;
+///
+/// let potential = Gaussian{a: 8.0, b: 0.5};
+/// assert_eq!(potential.energy(0.0), -8.0);
+/// assert_eq!(potential.force(0.0), 0.0);
+/// ```
+#[derive(Clone, Copy)]
+pub struct Gaussian {
+    /// Distance constant of the Lennard-Jones potential
+    pub a: f64,
+    /// Energy constant of the Lennard-Jones potential
+    pub b: f64,
+}
+
+impl Potential for Gaussian {
+    fn energy(&self, r: f64) -> f64 {
+        -self.a * f64::exp(-self.b * r * r)
+    }
+
+    fn force(&self, r: f64) -> f64 {
+        2.0 * self.b * r * self.energy(r)
+    }
+}
+
+impl PairPotential for Gaussian {
+    fn tail_energy(&self, _: f64) -> f64 {
+        0.0
```

Comment:
Is this temporary? It feels like this potential is not a long-ranged one. But I've never encountered it, so I don't know what it is used for.

Reply:
Good question :) For my use case the cut off distance is typically way larger than the potential's width. But you are right, I should implement it anyways.

Reply:
Just out of curiosity, what do you use this potential for?

Reply:
As an "external field". I use it to draw molecules to their reference positions in a crystalline phase. I want to use Lumol to compute free energies of solids. If you are interested in the method, I can give you more details.

Reply:
Yes, I am always interested! Will you then to implement another propagator for this, or can you re-use the existing ones?

Reply:
I might invest some time to bring basic free energy functionality to Lumol (lambda scaling) to make these computations more convenient.

Reply:
As a first step I use a simple MC algorithm where the particles that interact via Gaussians are frozen in place (no translation). But I think that your idea of an external field could also work.